tests/15-rpl-classic/code-rank-error/Makefile

```diff
@@ -0,0 +1,8 @@
+CONTIKI_PROJECT = simple-node
+
+all: $(CONTIKI_PROJECT)
+
+MAKE_ROUTING = MAKE_ROUTING_RPL_CLASSIC
+
+CONTIKI = ../../..
```

Comment:
The top Makefile defines the top directory, so I don't think `$CONTIKI` is required to be defined by test cases anymore.